### ISSUE TITLE
Updated package.json seems to drop a lot of preexisting contentsm, including dev scripts

### ISSUE THREAD

**Issue by strogonoff** [paneron/paneron#67]
The same commit that caused #65, https://github.com/paneron/paneron/commit/c0bbd326455809caee0b05ee2719a0db5dfbe11f, dropped `dev` script (for development purposes, so that one could run `yarn dev` or similar), as well as huge amounts of metadata that I presume was actually used by electron-builder.

Within that commit I am not seeing the metadata being moved to some other configuration recognized by electron-builder, which makes me suspect the builder won’t have the info necessary to build the project as before (but I’m still investigating).

(Filing as “bug” because missing dev script qualifies as a regression)

**Comment by strogonoff** [comment]
Some metadata that had been dropped:

<img width="504" alt="Screenshot 2022-12-17 at 18 06 05" src="https://user-images.githubusercontent.com/292959/208236647-7775279c-9944-4dec-a493-39c57ae79af4.png">

**Comment by strogonoff** [comment]
Ah. I see that those commands are part of @paneron/core.

**Comment by ribose-jeffreylau** [comment]
@strogonoff Yes, something like `@paneron/desktop` seems to be a natural next step.

**Comment by strogonoff** [comment]
I think we should still make sure documentation and build work before that is implemented, since monorepo switch is not in a branch anymore this technically becomes a regression…
